addon/ng2/commands/build.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var path    = require('path');
+var Command      = require('ember-cli/lib/models/command');
+var win = require('ember-cli/lib/utilities/windows-admin');
+var BuildCommand = require('ember-cli/lib/commands/build');
+var BuildTask = require('ember-cli/lib/tasks/build');
+var BuildWatchTask = require('ember-cli/lib/tasks/build-watch');
+var TestTask = require('../tasks/test');
+
+module.exports = BuildCommand.extend({
+  name: 'build',
+  description: 'Builds your app and places it into the output path (dist/ by default).',
+  aliases: ['b'],
+
+  availableOptions: [
+    { name: 'environment', type: String,  default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
+    { name: 'output-path', type: path,    default: 'dist/',       aliases: ['o'] },
```

Comment:
You are correct, this is a holdover from pulling in the ember-cli build command.